### PR TITLE
#167: add commits, hoc, last_commit to release-published fact

### PR DESCRIPTION
**Pull request by Yegorov**

### CODE REVIEW

**Comment by Yegorov** [comment]
@Suban05 can you review it, please?

**Review by Suban05 — COMMENTED** [review]
@Yegorov I have left a comment

File: judges/github-events/github-events.rb [hunk 1]
```diff
@@ -58,6 +58,28 @@ def self.fetch_contributors(fact, repo)
...
+    Fbe.octo.compare(repo, last, fact.tag).then do |json|
+      info[:commits] = json[:total_commits]
+      info[:hoc] = json[:files].map { |f| f[:changes] }.sum
+      info[:last_commit] = json[:commits].first[:sha]
+    end
+    info
+  end
+
+  def self.find_first_commit(repo)
+    commits = Fbe.octo.commits(repo)
```
[the remark this answers is not in the document]
Comment (Yegorov) [reply judges/github-events/github-events.rb:74]: @Suban05 @yegor256 Ok, but can we create a release without commits?

[the remark this answers is not in the document]
Comment (yegor256) [reply judges/github-events/github-events.rb:74]: @Yegorov good question. Most probably it's impossible. You can try: create a new empty repository and try to make a release.

[the remark this answers is not in the document]
Comment (Yegorov) [reply judges/github-events/github-events.rb:74]: @yegor256 It seems not
Release in empty repo

**Comment by yegor256** [comment]
@Yegorov thanks!

**Comment by 0crat** [comment]
@Yegorov Hey there, awesome work on your contribution! 🎉 You've snagged +25 points: 20 for showing up, and 5 for those 108 [hits-of-code](https://www.yegor256.com/2014/11/14/hits-of-code.html) you cranked out. Nice job keeping it under 200 HOC, avoiding any deductions there. Your total balance is now [+269](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Remember, more reviews and comments can boost your points, and [keep](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) those contributions coming – they're adding up nicely!

**Comment by 0crat** [comment]
@Suban05 🌟 Fantastic job on the review! You've earned **22 points**: 15 base + 7 for your insightful comments. Your dedication is paying off - your balance now stands at [357 points](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Keep up the great work!

**Comment by 0crat** [comment]
@yegor256 Thanks for your thorough code review! 🎉 You've earned +22 points: +15 base points, and +7 for your 7 insightful comments. This aligns with our policy of rewarding reviewers, especially those who provide valuable feedback. Your current balance stands at [-548](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Keep up the great work in future reviews!

**Pull request merged**